modules/social_features/social_search/config/install/views.view.search_all.yml

```diff
@@ -42,15 +42,17 @@ display:
           sort_asc_label: Asc
           sort_desc_label: Desc
       pager:
-        type: mini
+        type: full
```

Comment:
@jochemvn was this intentional?

Reply:
yes. the all search pager was inconsistent with the other searches